code/modules/hallucinations/effects/blind_rush_hallucination.dm

```diff
@@ -0,0 +1,69 @@
+/**
+  * # Hallucination - Blind Rush
+  *
+  * Makes target blind and causes them to see semi-transparent humanoids running at them.
+  */
+/datum/hallucination_manager/blind_rush
+	initial_hallucination = /obj/effect/hallucination/no_delete/blind_rusher
+	trigger_time = 34 //total length of the hallucination is a little more than ten seconds
+
+
+/datum/hallucination_manager/get_spawn_location()
+	var/list/turfs = range(13, owner.loc)
+	return pick(turfs)
+
+/datum/hallucination_manager/blind_rush/on_spawn()
+	owner.playsound_local(get_turf(src), istext('sound/spookoween/ghost_whisper.ogg') ? get_sfx('sound/spookoween/ghost_whisper.ogg') : 'sound/spookoween/ghost_whisper.ogg', 30, TRUE)
```

Comment:
Why are you doing this stuff with `istext`?

Reply:
This is how I've seen it done everywhere else I looked, is it unnecessary?